components/infoColumns/index.tsx

```diff
@@ -98,57 +98,65 @@ const InfoColumns = () => {
           post-graduates, and professionals from backgrounds in academia,
           finance, politics, business, technology and more to deliver a
           high-caliber educational experience to our delegates.
+          <br />
+          <br />
+          This coming year, we welcome you - delegates, faculty, sponsors, and all others -
+          <em> in-person</em>, to the <strong> 18th Chicago International Model United Nations
+          conference</strong>. We hope you can make it!
         </Body>
       </CardContainer>
-      <CardWrapper flex="2" margins="1rem 0" pad="1rem 0 1rem 1rem">
+      <CardWrapper flex="2" margins="1rem 0" pad="1rem 0 6rem 1rem">
```

Comment:
Does this bottom padding still look good when the two cards are on top of eachother screen widths 1000 - 14000?

Reply:
Looks fine to me I think